kde-frameworks/kdelibs4support/kdelibs4support-5.109.0.ebuild

```diff
@@ -76,6 +74,13 @@ RDEPEND="${COMMON_DEPEND}
 BDEPEND="
 	dev-lang/perl
 	dev-perl/URI
+	handbook? (
+		app-text/docbook-xml-dtd:4.2
+		=kde-frameworks/kdoctools-${PVCUT}*:5
+	)
+	network-status? (
+		net-misc/networkmanager
```

Comment:
BDEPEND-only looks suspicious for something which provides a binary that's useful at runtime and a library.

Reply:
That would make it `DEPEND`.